lib/sequel/extensions/pg_triggers.rb

```diff
@@ -329,6 +331,14 @@ def pgt_trigger(table, trigger_name, function_name, events, definition, opts={})
       def pgt_mangled_table_name(table)
         quote_schema_table(table).gsub('"', '').gsub(/[^A-Za-z0-9]/, '_').gsub(/_+/, '_')
       end
+
+      def pgt_pg_trigger_depth_guard_clause
+        <<-SQL
+        IF pg_trigger_depth() <> 1 THEN
```

Comment:
This doesn't use the `:prevent_depth` option.  Something like ` pg_trigger_depth() > #{opts[:prevent_depth]}`, pass the `opts` to `pgt_pg_trigger_depth_guard_clause`. and add the conditional inside `pgt_pg_trigger_depth_guard_clause` so you don't need it in all callers.

Reply:
Done